src/source.rs

```diff
@@ -63,6 +63,11 @@ pub trait Loader {
   fn get_cache_info(&self, _specifier: &ModuleSpecifier) -> Option<CacheInfo> {
     None
   }
+  /// An optional method which returns the JSX import source module which will
+  /// be appended to any JSX import source pragmas identified.
+  fn jsx_import_source_module(&self) -> &str {
+    "jsx-runtime"
+  }
```

Comment:
I'm not sure about this being on the loader trait. Seems more like an option that can be provided when creating a graph. That way someone doesn't need to provide a different loader based on what `jsx_import_source_module` string they want to use.

Reply:
Well, I don't want to start a slippery slope of config for the graph. Maybe it isn't the loader trait, but actually the resolver trait though now that I think of it and the lack of a resolver will default to this default.

Reply:
Ok, I moved it to the Resolver trait and I like it better, as it removed it from `parse_module`.  It only really comes into play in very narrow situations, basically when you want to support the development mode for JSX transforms, which really is only a runtime thing, not really any other point in time. I really don't want to do it all.

Reply:
Yup, resolver trait is better!